src/backend/browser/sandbox.ts

```diff
@@ -167,29 +169,48 @@ export const api: Sandbox = {
       ],
     });
   },
-  showMessageDialog(obj: {
+  async showMessageDialog(obj: {
     type: "none" | "info" | "error" | "question" | "warning";
     title: string;
     message: string;
   }) {
-    window.alert(`${obj.title}\n${obj.message}`);
-    // NOTE: どの呼び出し元も、return valueを使用していないので雑に対応している
-    // eslint-disable-next-line @typescript-eslint/no-explicit-any
-    return Promise.resolve({} as any);
+    const { promise, resolve } = Promise.withResolvers<void>();
+    Dialog.create({
+      component: MessageDialog,
+      componentProps: {
+        type: obj.type,
+        title: obj.title,
+        message: obj.message,
+      },
+    }).onOk(() => resolve());
+
+    await promise;
+
+    // 誰も使ってないので適当な値を返している。型定義をPromise<void>に変えてもいいかも
```

Comment:
やっぱり消していい気がしますねぇ！！

Reply:
あっ　と思ったけどこれ返り値が使われてないという意図なんですね！
ちょっと目的を足しときますか！
```suggestion
    // 誰も返り値を使ってないので適当な値を返している。型定義をPromise<void>に変えてもいいかも
```